ci-operator/config/kubevirt/node-maintenance-operator/kubevirt-node-maintenance-operator-master.yaml

```diff
@@ -37,11 +38,14 @@ tests:
 - as: openshift-e2e
   steps:
     cluster_profile: aws
+    dependencies:
+      OO_BUNDLE_IMG: nmo-bundle
     env:
-      OO_CHANNEL: "4.8"
       OO_INSTALL_NAMESPACE: default
-      OO_PACKAGE: node-maintenance-operator
-      OO_TARGET_NAMESPACES: '!install'
+      OO_TARGET_NAMESPACES: OwnNamespace
+    pre:
+    - chain: ipi-aws-pre
+    - ref: optional-operators-ci-run-bundle
```

Comment:
Please remove the modification of the existing job: we only use that one for testing, but we definitely do not want  to actually mess with someone's CI jobs.

Reply:
Would it be ok to leave it here for now for testing purposes and remove the commit that changes the CI job before merging the PR?

Reply:
Yes, absolutely. Would it be ok to keep this for testing purposes and remove the modifications before merging the PR?

Reply:
Sure, we can keep the change until we're happy with the PR and only remove it just before merge.